### PR TITLE
Fix doc for MultiHeadAttention's output_shape arg

### PR DESCRIPTION
**Pull request by tsdeng**
Fix doc for MultiHeadAttention's output_shape arg. Instead of projecting to key feature dim, the code actually projects to the  query feature dim.
This is to be consistent with the doc on the return arg and the actual code.

### CODE REVIEW

File: keras/layers/attention/multi_head_attention.py [hunk 1]
```diff
@@ -187,8 +187,8 @@ class MultiHeadAttention(Layer):
       dropout: Dropout probability.
       use_bias: Boolean, whether the dense layers use bias vectors/matrices.
       output_shape: The expected shape of an output tensor, besides the batch
-        and sequence dims. If not specified, projects back to the key feature
-        dim.
+        and sequence dims. If not specified, projects back to the query feature
+        dim(the query input's last dimension).
```
Comment (fchollet) [inline keras/layers/attention/multi_head_attention.py:191]: Please add space after `dim`

**Review by fchollet — COMMENTED** [review]
Yes, this is right. Thanks for the PR.

**Review by fchollet — APPROVED** [review]
LGTM

**Comment by gbaned** [comment]
Hi @tsdeng Can you please resolve conflicts? Thank you!

**Comment by ianstenbit** [comment]
@tsdeng please take a look at the linter error -- thank you!

**Review by ianstenbit — APPROVED** [review]
LGTM -- thank you!

@fchollet are you able to merge (I don't have that power)

**Pull request merged**